app/templates/mascots/payment.hbs

```diff
@@ -1,50 +1,312 @@
 <section class="container layout">
   <div class="lg:col-4">
-    <h1>Tomster and Zoey Commission Payments</h1>
-
-    <div class="left-col">
-      <div id="wufoo-q3qhbsp095gijb">
-        Fill out my
-        <a href="https://tilde.wufoo.com/forms/q3qhbsp095gijb">
-          online form
-        </a>.
-      </div>
-      <script type="text/javascript">
-        var q3qhbsp095gijb;
-        (function(d, t) {
-          var s = d.createElement(t),
-            options = {
-              'userName': 'tilde',
-              'formHash': 'q3qhbsp095gijb',
-              'autoResize': true,
-              'height': '646',
-              'async': true,
-              'host': 'wufoo.com',
-              'header': 'show',
-              'ssl': true
-            };
-          s.src = ('https:' == d.location.protocol ? 'https://' : 'http://') + 'secure.wufoo.com/scripts/embed/form.js';
-          s.onload = s.onreadystatechange = function() {
-            var rs = this.readyState;
-            if (rs)
-              if (rs != 'complete')
-                if (rs != 'loaded') return;
-            try {
-              q3qhbsp095gijb = new WufooForm();
-              q3qhbsp095gijb.initialize(options);
-              q3qhbsp095gijb.display();
-            } catch (e) {}
-          };
-          var scr = d.getElementsByTagName(t)[0],
-            par = scr.parentNode;
-          par.insertBefore(s, scr);
-        })(document, 'script');
-      </script>
-    </div>
+    <h1>Tomster and Zoey Commission Payment</h1>
+
+    <p>This form is processed using Stripe.</p>
+
+    <p>This form is only for Tomsters that we've already discussed payment and details on. If you have a new idea, <LinkTo @route="mascots.commission">reach out to discuss</LinkTo> first. 🙂</p>
+
+    <form
```

Comment:
How to create the HTML markup for `<form>`:

1. On Chrome, isit https://tilde.wufoo.com/forms/q3qhbsp095gijb and inspect the form element.

2. Right-click the element, then select `Edit as HTML`. This allows us to copy the HTML code.

    <img width="600" alt="Copy the HTML code for the form element" src="https://user-images.githubusercontent.com/16869656/98418384-5faf7300-2048-11eb-9e6c-034fb3098219.png">

3. Use [Prettier playground](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAeAZhATgWwAQCWAJgLwA6ImuAzAKwV5QCGOc5l2OdDYANkwGcB7AO4BXTBDwwIABwAyTAEZxeeWUwDmcAIwMmYMHFkwAtGAAWTLALgx2AVQAqAMVMAOfWJmQcs3nZsFBDo6AwIYDAAnrJBIDhivDAEGlgwAPRUOKZETDBMDKwwFhCkFLIQAjAMUBAAbky8xHlx+pEE0OwWMDCyAkjp6cm8RHAAdOKSY76ZnALpAI7UCxZKArIADACctJoEAFZKFAB8ZFBnqBZwTKNYhGUgVzdwWDz8QuwEUJgnZ2QwlwATHg+IJhBQTk4IDgqi88EwoEQ8AAtCBwKJ4ADC0JwBCEHSgeAACkwoqxYAJUOkLIDTucAUQCHUQe9wSBIRY8XgsoQBOosBAjEI4EixAIvpo8ABlGBYFLjVBKLB0xXKs5OTl8nlc6C8DFUPBQmHwGzSKwwPAiOAAcjqcHhvCw1yIGMZAjAYtsSI0ZIQFoRSNG+QIvD50DGeAAkug8FEIGI8FY7fDGHARPdrgAaPBOgwWPDxi0yPBuj1CbkEGwWwC8G4AhPapjLqKupzpedL+1SgqES7eqnf+HYuTXu7EwdCaehALLB7FqsoRAn48DwK9Xv3pqH4KjUoI+FFG7oYxHYyRgAUnvf+ADkWHBB33-qh1giRxQnQsAPqT6d7kDvk4AFRUs+UCXn2VJbqovZPhooH3qgXyyN4r4gC4BCqEQ37MKw7BoRh37RLEJ5wAAHtUU67my6DoSM0ikRa6DnFODS8GIrRTuKABecSeFO+RKF8owkewGwMNAADW6JiLIXQBgEkZQEhMAABTFHiACUADcYlQJYCLaLJiLyYp3iqZqWkMP4BhwCUIwvOwDDvmIlYig5IBgZuyiqNy2C4TRmEnGhVYQV5vDQekIHQZF8GIchx4UHhIyAjUt5+RhyV8TEcTwGRbwzhQ1EYXRZF4LwTF4CxbFuXgXFxDoAAsDD8YJpEiTpklRNJhlEMZSlmRp2lTtAelQAZFBWEZcAKX1akCBZU5WUYtm3NVTkuQ864DgCkFqFQaVJScihVCF27hdFG7pE07bnDdm4EChY60E0WwMLk+SmJMEAQKYhUjNVb1MD9-mmIR2X0Xlv5zlgC5Lvaq4rpt22hT+bIHmAR4PKeAQvVOe0Jf5ONgdifgIlE96PpdoXQY2HmxRa8WoQTKU4fjGE49IWXEblFGsqO-nFRarCMmIOAMJV7F4DgTAkQEo3FOwgK0PQfHKC1wkUKJQ1QB1XXjXJU0mSps1aXgOkjWNjz69NpnG4N6j8EtEB2VgblgRTNMdhBBDXX8XbDgzj1NNQENslDMMtPDCPuZ7O0o+waMYyeBBnnAwe475rMjGnYEAKJSyG5PgSBKHvh+adx2+cALIBwGwW7AKU6dnse-Sj50yhiVEOX2FxJ35eg+wcD57wDDrKovCWHAYDiaOjS2CHfNFTlgsigQIti40VUQlOUsywgmjyxQivK9IquIq1GvtVJMl65N1tG+ZdvDRNFsTT1BszY-lkOzZTsrdvOYq7OSdBtaOF0W4Uyur7X2d0HoQHHAQHQ34AYfQkF9IGGF-p5EBr9IgINOYUGXgvCgYcoCLgjpHRGnltwVxAAnKcDMsa6G-HjRm+ELz3gAOqcksIaaEsI7gBnSNgFEaIMRclmuoUk5IGLYAAPyF0fMXBmpckHEL-FXGuEU66FxOlBZuTJaaGw7v5NRU4e77UwgRAhIAiE83ypQfmy9Jar3XsxTeEtd6ywPhYBWSsmpnyEm1LWOsb6Wzvobfqc1NKmy1ubOIb9eo2y-lONaIDqqLV-s7V2uj0gQIblAm6MD-YPEDgQRq9jIYQHnGQ2GkdYnQVjpReOcBDwMMxsnAIFSiY4jxOKaAeAACCOB4ywEUQCZRDxS4VNof+dyQFtEInrnosKnti7NIoAIMkSgnYnAACS1yWWsnRrcATtwZp3GZFjM5EBmQPQh4NKlUScfREEYgsBOl0hiWo6wvgb1YhLWq7AdCa1PgJc+6sQCgoktfbqSSH4DTNi-BJVtIm20ckA9aOTTnUO8qwy5JwAAiTt+A2BWWdE5W0YIvg2SALZOAdlqGhoyEiJwxiHLghudZvMKBGApIjNuxiLn+XqqYLCqUbmiusURB53NaG4IFowSosg-nuIBdVIFR9CjS28YfEAGUwVq2CQWbWsLb7v3vlE+aJr4lwo-skxFYCqWx3xSKsVJxMR+kpI3fRF1zqQO9tA26JTRzwKeoggAbOoyhTqAS4NsDAaCABCUwpgADaBAYxJsjDnPAAAfPAykD72hzXgdw6kAC6qbGlwG0IiFCTCdBRqeS090SqankPgAKmAkYQQ0ugNRXAZo8hRkTEwZMjRcwuhqgQTQUARRmntPIAgfLxSjXhJoJ0Q8-SjpEIIfkcA6gdDFHqGqO6j1MDwAACXHhAKUs6oBjHGdSwkKiq5fmbbMzR8yOXLMurWhARBk2prTYBzNVbTDAfTZm0qFpS3uArY05GjDOm6E-bS+hYFe1gH7d8Ss+Bigjt7Ume0k7nQYnFHOhdxQl0roQGuyUWgt0yN3fu2QToj3xkXBR89BBL03t4LwO9D6n2nKpZM9gqj0M8o0dXH9izOVbUgVTT2SbQOInA9WgxTZ4LcocQIMQShcFUPOQ8TuTbma91MZ++54ArjTx2ayltBV+aTwcxAJzFUPHsGxHh3AIpszJ2tHyFQCAZ1UaIEmgJ4KgmXziciu1lr0WxtxTuGTlgIB0YYK6-CzaPJ6d-BlujH4PzONIZ2u87lfODqFoFmAwW8ChcJJR+dkXf3jKUbBWhRWjAlYWNEbtSmHwN39QUlTfrKVFy67SgzRn-ImaFWZ0xAB2SzlidCrcyjKuzU9xKOfUQqtze2PP-K3iAYj46EDWhgNmAQUhe0CBKIkIgUBruNftLN3EPQJRmi5D6FjUQ7BjGB9Fo1cWbUJfNfCq1g0kM0NpT1yrPkXY3I292l93WSjFdK688rsMTgXbtK9m7NV7s1SeyMYn72aqGa+8kddkj-s7sBzAYH7KFP13E9N9LWPesfn62TWNQ2hvtYujtClRycUFbZLN4zKXTPrd4owCVbCRg6CV7Zo7+3nOOKKlrk7aqzsaj+9IndXJIB2jlOukI1EwB8bUOxiA+wp4WgaHKBEMABC3akLUUqdGyE-aY3AbdsBeSphFCKJ9KsYsXyhUi-SKKImf0dXD7yCPedI5y2r3i+XucOMRyVsr1Toa1JaByE3vpQ-m-qC8H7NuV32-5E7l3Xn3cUm90qv3q7A+buDyxrk86I9EHZ6N4XEy8+FYz31gbQvhsi4557H1qyJuS6pXkwxi-412HCoU32psin9m2vdWlShvAyDIQ0+C+Sx8i-brZzkRBRjlWueAd5nyYAkm0InXlb+-Sf6R+LOwPVAALI5ziDkKPaRgwDLY6BXpXpOC8D1QABKcB4gAAQnAPsAAGp1D1TLboD1TgHvAWBEBEjoFwAOCEoWCcK8DUBMDuBIJShgBRA6AiDsD1zwxnCcHDbcFbS8F9j1IIxz48GrgISLbsACCXYuCcBrabJSEyE65KAwBOAlSfbJxNQ2JqHkRebqoUDyB2ANbEqGiahRZawjJigigQAiBQDxx3q07Jw5xE6e7KTzSc4PhiaEhrhibgTr46Zi6BoH6PjDi0oP6VYeQuoZzgA4h+hEpSBXjVJ4BoSCbGFcgADy3g5K8Ey81g1wyuLMUROAMi3+BRRRqSVhbI34kAoYwKJwVI2RuYRiSkHM22D+T+xRxAVQLAsgshIAHR+Qfgp2cQTAnEFgsgnC4kMAJESBIggIRIUQvACwIgUoWBYgwBAAGgAIokDsExTGL37EBtHtKDy6RYAxDwCkFggiDYBEA9ERCnEmAigkhCBXFYA3GG7sRuFiFNH7GP4IDFGdGe49EAkCCDHsDAAABkCwYg1SmkLwAoNgkJ0JMAmkSAGwmYiJMJnRaQTgBArAGJyJqJ6JUJMJgGOJeJxJBJaJ+JmkTo6ALwLw1JSAUAiQvAAAvoNuBHfjYq0X8Ucbyk0NPKkVgDnLAC8D0XwCuuJEKSKSaKCdvH+rvg3D2BcLMLgMcCAJmCAHIMkNAAIMgKANYAKCICSJ8nqSgI0HulEHqZqUqAYJJDAFKBoHbqNMgLKGxJqUPCoL8UQIoKNGIFoHANIbgHkPTpoMgCAEwN4BABqY8DADgLwNwsnK0k6XAFKPRsnEyMnFEOGWAEIDGV8LYGkESNDJoFLMgOgHPHAJqfsAICRGgdDNPHYFKLeMuvOuWZWdWbWfeqNAEBsUiXAO2aGFWSAKkIWeGd0PGTGexl8DAJwsQMUMgO4GiSOQKLYJwtDLIOGexq0i8HaDGRSXAMWXIGaRGQIKYIPqMEQDGWkoeSWWWUgBWUOZqbYLiK6VgO6XShKL2f2YObYJqfxHOUQAuUgICP+dDCGBKMTPeSAK0rQDGRYU4MoGaY+X+SAHUGxApE-g6WAHKCYIMoiDKPMQOQ+ZWayayUAA) to fix indentation issues. Make sure to select the `html` option for `--parser`.

    <img width="1677" alt="Screen Shot 2020-11-06 at 3 57 10 PM" src="https://user-images.githubusercontent.com/16869656/98418548-bf0d8300-2048-11eb-8f4f-77a3f18074cf.png">

4. Copy-paste the Prettier-ized code to `app/templates/mascots/payment.hbs`.

5. Add the `.tomster-form` class to the `<form>` element to get _most_ of the styling right.

6. Update the HTML markup and stylesheet to get the _rest_ of the styling right.